src/rustdoc/meetings.md

```diff
@@ -0,0 +1,13 @@
+# Meetings
+
+The rustdoc team hosts a meeting every second monday on each month on the 
+[t-rustdoc/meetings channel on zulip](https://rust-lang.zulipchat.com/#narrow/channel/393423-t-rustdoc.2Fmeetings)
+at 21:00 CET (UTC+1) and CEST on summer (UTC+2).
+
+A new thread is open a month ahead of time to remind people who want to attend about the time
+and the agenda.
+
+Anyone is open to participate to the meetings.
+
+If you want to add items to the agenda, please comment on the zulip thread for the next meeting
```

Comment:
(Side note) If we go down the route of discouraging non-members to edit the agenda themselves directly, we should (sooner than later) migrate to a HackMD "team account" setup (disclaimer: I don't how this works) since nobody will be aware of this change even if we make a Zulip PSA (which would be kinda weird anyway).

I think we briefly touched upon that during AllHands '25. Anyways, this can/should be done separately from this PR.